triggers/service/auth/src/main/scala/com/daml/oauth/middleware/Config.scala

```diff
@@ -63,6 +68,64 @@ object Config {
           if (x.isEmpty) failure("Environment variable DAML_CLIENT_SECRET must not be empty")
           else success)
 
+      opt[String]("auth-jwt-hs256-unsafe")
```

Comment:
Maybe worth factoring out similarly to how we factor TLS options.

Reply:
Good idea, I'll do that in a follow up PR.